docs/Thana.md

```diff
@@ -0,0 +1,45 @@
+# تجربتي مع موقع Crowdin 
+ 
+ تمّ إنشاء مجتمع العلوم المفتوحة في السعودية OSCSA تماشيًا مع رؤية السعودية 2030 لدعم توفير بنية تحتية للعلوم المفتوحة في العالم العربي من خلال العمل مع مجتمعات العلوم المفتوحة الأخرى مثل  Turing Way  باستخدام  GitHub ومنصة Crowdin
+موقع[ Crowdi](https://crowdin.com/)n عبارة عن منصة تعريب تقدم خدمة ترجمة المواقع إلى عدة لغات بشكل احترافي وجماعي دون ضياع الجهد أو تداخله عن طريق تقاسم المهام بين المترجمين.
+
+ضمن مساهمتي في [Outreach](https://outreachy.com/)y  من خلال اختيار مشروع مجتمع العلوم المفتوحة في السعودية، وبعد الاطلاع على ملف التعليمات:
+1. قمت بإنشاء حساب على منصة Crowdin عن طريق التسجيل من خلال البريد الإلكتروني. 
+
+![image001](https://user-images.githubusercontent.com/31999574/199712779-2d9516a5-c475-4dc0-a330-785aedf2c5be.png)
+
+2. يتمّ إرسال رسالة تأكيد إلى البريد الإلكتروني لتفعيل الحساب الجديد على الموقع.
+
+1. بعد تسجيل الدخول، 
+تظهر الصفحة الرئيسية للموقع (قمت بطلب الانضمام إلى مشروع Open Life Science Cohort 3 Videos and Subtitles بعد النقر على الرابط المرفق في ملف التعليمات)  لأصبح عضوًا في المشروع بعد موافقة المسؤول طبعًا.
+ 
+![image003](https://user-images.githubusercontent.com/31999574/199713362-c5dc7682-f148-4225-8f4d-4961d0ecd588.png)
+
+1. يمتاز الموقع بواجهة سهلة وبسيطة الاستخدام، ومريحة للعين. 
+            تظهر الإشعارات لديك بالنقر على أيقونة الجرس التي تظهر أعلى اليمين.
+1. يمكن التعديل على بياناتك الشخصية من خلال خيار إعدادات ( setting)، أعلى يمين الصفحة الرئيسية كتغيير الصورة الشخصية أو تغيير الاسم الأول والأخير وغيرها من الخيارات. 
+ 
+![image005](https://user-images.githubusercontent.com/31999574/199713480-00146e0a-5bab-4402-a20c-21287d1a9701.png)
+
+1. يدعم الموقع العديد من اللغات، قمت حسب تجربتي بالترجمة من الإنجليزية إلى العربية.
+
+المهمة الموكلة إلينا هي أن يقوم **كل شخص باختيار ملف ليقوم بترجمته،وهذا الملف هو ترجمة لفيديو معين قمت باختيار ملف Open-Data.srt إلى العربية وقام باقي زملائي باختيار ملفات أخرى**.
+
+# عند فتح الملف المراد ترجمته، تنقسم النافذة إلى ثلاثة أقسام رئيسية:
+![image007](https://user-images.githubusercontent.com/31999574/199715144-83da7733-b8fc-46d6-8d12-22d84e923a90.png)
+* [ ] الجزء الأوسط (2) وهو الجزء الرئيسي، حيث نختار أفضل ترجمة من الأسفل (وهي مترجمة ترجمة آلية)، ونقوم بالتعديل عليها أو إعادة ترجمتها كليًا حسب ما يقتضيه النص بالإضافة إلى الاهتمام بأدوات الترقيم ثم اختيار حفظ ليتم إضافة هذه الترجمة واعتمادها (كما في الصورة المرفقة). 
```

Comment:
This paragraph is aligned left-to right, it should be right-to-left, can you fix it, please?

Reply:
I think it's good now